docker/pravega/scripts/common.sh

```diff
@@ -36,7 +36,7 @@ add_system_property_ecs_config_uri() {
 
 # Add ECS certificates into Java truststore
 add_certs_into_truststore() {
-    CERTS=/etc/ssl/certs/java/ecs-certs/*
+    CERTS=/etc/secret-volume/ca-bundle/*
```

Comment:
Curious, what is this `ca-bundle` directory under `/etc/secret-volume`? It's not clear to me from the PR description. Is it the certificate of the CA that was used to sign the ECS server certificates?

Could we use a more specific name? Not sure what the ca-bundle represents exactly.

Reply:
Ok, I found the context in the corresponding issue: #4642. I'd suggest you add a bit of context to the PR description too, for future reference.

Reply:
PR description updated.